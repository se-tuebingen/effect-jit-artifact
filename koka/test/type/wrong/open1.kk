open type test {
  X
}

extend type test1 {
  Y
}