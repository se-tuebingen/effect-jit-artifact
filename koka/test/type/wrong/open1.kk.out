test/type/wrong/open1@kk(5,13): kind error: Type test1 is not defined
 hint: bind the variable using forall<test1> ?
test/type/wrong/open1@kk(5,13): kind error: Type test1 is not defined
 hint: bind the variable using forall<test1> ?